com.endlessm.HackSoundServer.json.in

```diff
@@ -6,7 +6,9 @@
     "sdk": "org.gnome.Sdk",
     "finish-args": [
         "--share=ipc",
-        "--socket=pulseaudio"
+        "--socket=pulseaudio",
+        "--socket=session-bus",
```

Comment:
I am not happy with this. According [documentation](http://docs.flatpak.org/en/latest/sandbox-permissions.html) 

> Access to the entire bus with --socket=system-bus or --socket=session-bus should be avoided, unless the application is a development tool.

Reply:
I think it's OK to have this, since we need it for the bus watcher.